robot_controllers/src/parallel_gripper.cpp

```diff
@@ -69,15 +110,36 @@ int ParallelGripperController::init(ros::NodeHandle& nh, ControllerManager* mana
   server_->start();
 
   // Get Params
-  nh.param<double>("max_position", max_position_, 0.1);
-  nh.param<double>("max_effort", max_effort_, 10.0);
+  if(!nh.hasParam("max_position"))
```

Comment:
none of these changes in this section are needed -- it is not standard convention to warn on unset parameters that have defaults, please revert this